warehouse/templates/pages/help.html

```diff
@@ -864,11 +864,11 @@ <h3>{% trans %}Resources{% endtrans %}</h3>
 
       <h3>{% trans %}Contact{% endtrans %}</h3>
       <p>
-        {% trans trimmed pypa_href='https://pypa.io', irc_href='https://web.libera.chat/#pypa', discuss_href='https://discuss.python.org/c/packaging/14', title=gettext('External link') %}
+        {% trans trimmed pypa_href='https://pypa.io', irc_href='https://web.libera.chat/#pypa', discourse_href='https://discuss.python.org/c/packaging/14', title=gettext('External link') %}
           The <a href="{{ pypa_href }}" title="{{ title }}" target="_blank" rel="noopener">Python Packaging Authority (PyPA)</a>
           is a working group who work together to improve Python packaging. If you'd like to get in touch with a core packaging developer,
           use <a href="{{ irc_href }}" title="{{ title }}" target="_blank" rel="noopener">#pypa on IRC (Libera)</a>,
-          or <a href="{{ discuss_href }}" title="{{ title }}" target="_blank" rel="noopener">browse the online board</a>.
+          or <a href="{{ discourse_href }}" title="{{ title }}" target="_blank" rel="noopener">browse the online board</a>.
```

Comment:
here

Reply:
Or should we keep the same?

Reply:
In this context, Discourse and discuss are the same, so whichever.

Reply:
Thanks